integration-tests/multitenancy.test.ts

```diff
@@ -146,52 +146,54 @@ if (process.env.MULTI_TENANCY_ENABLED === 'true') {
             client = await getFhirClient('fhirUser user/*.*', true, { tenant: 'tenant1' });
         });
         test('requests without /tenant/<tenantId> in path should fail', async () => {
-            await expect(client.get(`${process.env.API_URL}/Patient`)).rejects.toMatchObject({
+            await expect(client.get(`${process.env.SMART_SERVICE_URL}/Patient`)).rejects.toMatchObject({
```

Comment:
Where is the value set for SMART_SERVICE_URL?

Reply:
Here: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/502c1a06155bd595cbb33821d1cb3b22429b1655/.github/workflows/deploy-smart.yaml#L198